src/orca/services/synapse/ops.py

```diff
@@ -28,3 +28,18 @@ class SynapseOps(BaseOps):
     client_factory_class = SynapseClientFactory
 
     client: ClassVar[Synapse]
+
+    def monitor_submission_view(self, view_id: str) -> bool:
+        """Monitor a submission view for completion.
+
+        Args:
+            view_id: The ID of the view to monitor.
+
+        Returns:
+            True if there are new "RECEIVED" submissions, False otherwise.
+        """
+        received_submissions = self.client.tableQuery(
```

Comment:
The most annoying part about views in Synapse in general, is that when you query them, it triggers an indexing job.  So here's what happens (at least when I was working with them)

1. Submission view is empty
2. Someone submits
3. You query the submission view -> it still shows up as empty, BUT the backend triggers an indexing of the view.
4. you query the view again -> the new submission shows up

Reply:
So I should have it query twice every time? I have noticed a quirk where I have to refresh the page twice to see an updated status in the view...

Or maybe we just have it check very frequently (I think you said every couple of minutes previously) and it will still grab the submission pretty soon after they are pushed?

Reply:
I think option 3 makes the most sense, but I am getting this error when I try to use it:
```
SynapseHTTPError: 403 Client Error: 
User lacks READ_PRIVATE_SUBMISSION access to Evaluation syn51356905
```
Even though I have administrator access to the submissions and the project as a whole

Reply:
I updated the function to use `syn.getSubmissionBundles`. I will add a test before marking this "ready for review".

Reply:
Agree with your rationale here.